### DISCUSSION TITLE
Input as dictated text

### DISCUSSION THREAD

**Discussion by BigSwingingProp** [Yue-Yang/ChatGPT-Siri#117] [Ideas]
It'd be great to have a version that asks for the input as dictated text. It would also be great if it could speak the result.

**Comment by chrisa2210** [comment]
> It'd be great to have a version that asks for the input as dictated text. It would also be great if it could speak the result.

You don't need to. I named my short cut "can I talk to Lucy" (you can name it anything you'd like. Then I say "Hey Siri, can I talk to Lucy" and the response I input in the start of the Lucy shortcut says "Hi [name], this is Lucy. What can I help you with?

Siri then speaks the entire sequence of the shortcut. And u can customize the messages in the shortcut to have Siri (in my case, Lucy) to say whatever you'd like. 

So any time i want to talke to siri i say "Hey Siri", when i want to talk to lucy (custom gpt shortcut), i say "Siri" - "Can i talk to lucy" then i get the pleasant custom into from lucy dictated by a beautiful english accent via siri
